### DISCUSSION TITLE
KAMP PRINT_START macro sets low bed temperature before probing for mesh

### DISCUSSION THREAD

**Discussion by WayneManion** [Guilouz/Creality-Helper-Script-Wiki#660] [Issues]
I print some ABS and ASA in my K1. I have KAMP installed using the script. 

 I like to preheat the printer when I print these materials in order to reduce warping. I preheat the printer by setting the bed temperature to 100C for ten minutes or so before I send my print job to the printer. 

I preheated the printer today. I noticed that the script (presumably the START_PRINT macro in the KAMP folder) set the bed temperature to 50C before it probed for the bed mesh. That does not seem like the right thing to do. I thought about editing the START_PRINT macro in the Start_Print.cfg file in the KAMP folder. The file appeared to be set to read-only so I thought I should ask about it here before running wild. 

Any thoughts on this?

**Comment by WayneManion** [comment]
I figured out the problem. I was wrapping the KAMP START_PRINT macro with another macro and I was not passing the bed temperature parameter through properly. The KAMP START_PRINT macro then used the default bed temp value of 50 C during meshing.

**Reply by thewolfman56** [reply]
What was your solution?  I'm having the same issue and it looks like my start_print gcode in Orca matches the syntax from the klippy guide

**Reply by Krahembuhl85** [reply]
I'm in need of the same solution too, please
